test/unit_tests/apps/ccdb-landing-map/js/landing-map-spec.js

```diff
@@ -1,7 +1,7 @@
 import 'regenerator-runtime/runtime';
 import * as sinon from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/node_modules/sinon';
-import Chart from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/Chart.js';
-import landingMap from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/landing-map.js';
+import Chart from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/Chart';
+import landingMap from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/landing-map';
```

Comment:
I tend to leave the `.js` so if you search for the filename its references will come up in the search results, but we do it both ways so whichever is preferred in this context is fine.

Reply:
Done. I noticed some imports have that and some don't have the js file extension.
I feel that it is better to have the `.js` so it is easier to search.